k8sfv/run-test

```diff
@@ -10,7 +10,7 @@
 : ${FELIX_VERSION:=latest}
 #
 # What version of the k8s API server to use.
-: ${K8S_VERSION:=1.7.3}
+# Update: See FV_K8SIMAGE in the Makefile for version information.
```

Comment:
I think we should either (1) continue allowing `run-test` to have sensible defaults of its own, or (2) make clear that it requires these variables to be set in the environment.

(1) would mean something like `: ${FV_K8SIMAGE:=gcr.io/google_containers/hyperkube-amd64:v1.7.5}` here

(2) would mean something like

```
if ! test -n "${FV_K8SIMAGE}"; then
    echo FV_K8SIMAGE must be set to indicate the hyperkube image to use
    exit -1
fi
```

(and, in either case, similarly for the other variables affected by this change)

WDYT?  (I think I slighly prefer (2), to avoid duplicating the default values.)